src/libraries/System.Drawing.Common/src/System/Drawing/DrawingComWrappers.cs

```diff
@@ -66,6 +66,20 @@ protected override void ReleaseObjects(IEnumerable objects)
             throw new NotImplementedException();
         }
 
+        internal void GetIStreamInterfaces(Interop.Ole32.IStream stream, out IntPtr streamWrapperPtr, out IntPtr streamPtr)
+        {
+            streamWrapperPtr = GetOrCreateComInterfaceForObject(stream, CreateComInterfaceFlags.None);
+            Guid streamIID = IID_IStream;
+            ThrowExceptionForHR(Marshal.QueryInterface(streamWrapperPtr, ref streamIID, out streamPtr));
```

Comment:
Why not release `streamWrapperPtr` right here? It should not be a responsibility of the caller to release it.

Reply:
Once you do that, the method can return IntPtr instead of void.